mlflow/pipelines/steps/train.py

```diff
@@ -381,35 +410,71 @@ def render_schema(inputs, title):
             '<div style="display: flex">{tables}</div>'.format(tables="\n".join(schema_tables)),
         )
 
-        # Tab 4: Examples with Largest Prediction Error
+        # Tab 6: Examples with Largest Prediction Error
         (
             card.add_tab(
                 "Training Examples with Largest Prediction Error", "{{ WORST_EXAMPLES_TABLE }}"
             ).add_html("WORST_EXAMPLES_TABLE", BaseCard.render_table(worst_examples_df))
         )
 
-        # Tab 5: Leaderboard
+        # Tab 7: Leaderboard
         if leaderboard_df is not None:
             (
                 card.add_tab("Leaderboard", "{{ LEADERBOARD_TABLE }}").add_html(
                     "LEADERBOARD_TABLE", BaseCard.render_table(leaderboard_df, hide_index=False)
                 )
             )
 
-        # Tab 6: Run summary.
-        (
-            card.add_tab(
-                "Run Summary",
-                "{{ RUN_ID }} "
-                + "{{ MODEL_URI }}"
-                + "{{ EXE_DURATION }}"
-                + "{{ LAST_UPDATE_TIME }}",
+        return card
+
+    @classmethod
+    def _get_run_and_model_urls(cls, run_id, model_uri):
+        run_tags = MlflowClient().get_run(run_id).data.tags
+        databricks_workspace_url = run_tags.get(MLFLOW_DATABRICKS_WORKSPACE_URL)
+        databricks_workspace_id = run_tags.get(MLFLOW_DATABRICKS_WORKSPACE_ID)
+        if databricks_workspace_url and databricks_workspace_id:
```

Comment:
nit: consider `return None, None` early here if one of the condition is not True. This will makes the large chunk of code below less indented:)